### ISSUE TITLE
Add UI tests

### ISSUE THREAD

**Issue by dolby360** [dolby360/hex_color_displayer#36]
We need to think about how to package can be automatically tested. Right now [tests](https://github.com/dolby360/hex_color_displayer/blob/master/test/index.test.jsx) are not doing anything.

**Comment by dolby360** [comment]
@Tamir198 Can you take this?

**Comment by Tamir198** [comment]
Yes, one i will finish with my current pr( I should find time to it by the end of the week) and take it
